### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by mattskrobola**
Quick readme update, can someone take a look? : )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 hello nin world
 nut
 peepee smeller
+murky
```
Comment (bitGraf) [inline README.md:5]: very good addition. nice catch champ!

**Review by bitGraf — COMMENTED** [review]
@Nick-Trot can we get your input on this?

File: README.md (hunk 1 above)
Comment (mattskrobola) [reply README.md:5]: thanks Xp

**Comment by Nick-Trot** [comment]
google USS Liberty

**Comment by Nick-Trot** [comment]
wehn u avoid the VAC ban 🤡

**Pull request merged**